src/modules/file/file.usecase.ts

```diff
@@ -245,14 +246,22 @@ export class FileUseCases {
     return files.map((file) => file.toJSON());
   }
 
-  moveFilesToTrash(
+  async moveFilesToTrash(
     user: User,
     fileIds: FileAttributes['fileId'][],
     fileUuids: FileAttributes['uuid'][] = [],
-  ): Promise<[void, void]> {
+  ): Promise<[void, void, void]> {
```

Comment:
`await` the promise.all and change the return type to Promise<void> instead of <[void, void, void]>